deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -4,12 +4,19 @@
 - name: Add Jenkins Nginx config
   template: src=jenkins.conf.j2
             dest=/etc/nginx/sites-available/jenkins.conf
-  notify: 
+  notify:
     - Restart Nginx
 
 - name: Enable Jenkins Nginx config
   file: src=/etc/nginx/sites-available/jenkins.conf
         dest=/etc/nginx/sites-enabled/jenkins
         state=link
   notify:
-    - Restart Nginx
\ No newline at end of file
+    - Restart Nginx
+
+- name: Setup hourly docker-gc cron task
+  cron: name="docker-gc"
+        user="{{ jenkins_name }}"
+        special_time=hourly
+        job="/usr/bin/docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /etc:/etc spotify/docker-gc"
```

Comment:
Addressed in https://github.com/azavea/raster-foundry/pull/430/commits/f5c8156b43791ddcf25da73b83911d57f54daa4a.